units/initrd-battery-check.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Check battery level during early boot
+DefaultDependencies=no
+After=initrd-root-fs.target
+Before=initrd-parse-etc.service
```

Comment:
make this initrd-switch-root.service for now

Reply:
> make this initrd-switch-root.service for now

Thank you so much sir.